frontend/src/components/App/Settings/Settings.tsx

```diff
@@ -72,24 +72,36 @@ export default function Settings() {
           {
             name: t('translation|Theme'),
             value: (
-              <Select
-                variant="outlined"
-                size="small"
-                defaultValue={themeName}
-                onChange={e => {
-                  dispatch(setTheme(e.target.value as string));
-                  console.log(e, e.target.value);
+              <Box
+                sx={{
+                  display: 'grid',
+                  gridTemplateColumns: 'repeat(auto-fill, minmax(120px, 1fr))',
+                  gap: 2,
                 }}
               >
                 {appThemes.map(it => (
-                  <MenuItem key={it.name} value={it.name}>
-                    <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
-                      <ThemePreview theme={it} />
-                      {capitalize(it.name)}
-                    </Box>
-                  </MenuItem>
+                  <Box
+                    key={it.name}
+                    sx={{
+                      cursor: 'pointer',
+                      border: themeName === it.name ? '2px solid' : '1px solid #ccc',
```

Comment:
using hardcoded colors (#ccc) is not recommended, for a border color you could use `theme.palette.divider `